proposed/phpdoc.md

```diff
@@ -1220,18 +1220,19 @@ The @method allows a class to know which 'magic' methods are callable.
 
 #### Syntax
 
-    @method [return type] [name]([type] [parameter], [...]) [description]
+    @method <static|> [name]([type] [parameter], [...])[: return type] [description]
```

Comment:
IMO, at least in this case `<name>` would be even better:
> Unlike original BNF, angle brackets ("<", ">") are not required.
> However, angle brackets may be used around a rule name whenever their
> presence facilitates in discerning the use of a rule name. 

But honestly, I cannot really figure out the rules how PSR-5 mixes ABNF and PHP :confused: